vnc/vnc.c

```diff
@@ -1422,6 +1447,11 @@ lib_mod_set_param(struct vnc *v, const char *name, char *value)
     {
         v->delay_ms = g_atoi(value);
     }
+    else if (g_strcasecmp(name, "guid") == 0)
```

Comment:
This is the only API to give the modules general data.  We don't give the modules access to the entire xrdp internals.
It was not meant to be only user data.